internal/rbd/nodeserver.go

```diff
@@ -436,7 +436,13 @@ func (ns *NodeServer) mountVolumeToStagePath(ctx context.Context, req *csi.NodeS
 		err = diskMounter.FormatAndMount(devicePath, stagingPath, fsType, opt)
 	}
 	if err != nil {
-		klog.Errorf(util.Log(ctx, "failed to mount device path (%s) to staging path (%s) for volume (%s) error %s"), devicePath, stagingPath, req.GetVolumeId(), err)
+		klog.Errorf(util.Log(ctx,
+			`failed to mount device path (%s) to staging path (%s) for volume 
```

Comment:
Prefer to use `"strings"+"and"+"them"+"together"` instead of a multi-line \``string with many ________ spaces`\`